modules/javafx.graphics/src/main/native-glass/gtk/glass_window.h

```diff
@@ -126,6 +129,7 @@ class WindowContext : public DeletedMemDebug<0xCC> {
     virtual void set_title(const char*) = 0;
     virtual void set_alpha(double) = 0;
     virtual void set_enabled(bool) = 0;
+    virtual void set_system_minimum_size(int, int) = 0;
```

Comment:
Maybe `set_headerbar_mininum_size()` ?

Reply:
Hmm... but it's not the minimum size of the header bar, it's the minimum size of the window.